packages/web/src/features/translation/VerseSelector.tsx

```diff
@@ -34,22 +34,26 @@ export function VerseSelector({ verseId, onVerseChange }: VerseSelectorProps) {
   };
 
   return (
-    <div className="flex flex-row gap-4 items-center">
+    <div className="flex gap-4 items-center flex-row">
       <TextInput
         name="verseReference"
         autoComplete="off"
         placeholder={generateReference(verseInfo, t)}
         onKeyDown={onKeyDown}
         arial-label={t('select_verse')}
       />
-      <button onClick={() => onVerseChange(decrementVerseId(verseId))}>
-        <Icon icon="arrow-left" />
-        <span className="sr-only">{t('previous_verse')}</span>
-      </button>
-      <button onClick={() => onVerseChange(incrementVerseId(verseId))}>
-        <Icon icon="arrow-right" />
-        <span className="sr-only">{t('next_verse')}</span>
-      </button>
+      {/* Use flex-row-reverse on rtl, so that the previous button is always
+          to the left of the next button. */}
+      <div className="flex ltr:flex-row rtl:flex-row-reverse gap-4">
```

Comment:
In a RTL language, we want the next button to be on the right. It's confusing for us, but natural in those languages. We should be able to remove this secondary flex box and leave it in the natural rtl order, but flip the icons.

Might be easiest to render two icons and show/hide the correct one with tailwind rather than trying to manage that with react

Reply:
I wondered if this was the case or not

Reply:
It just looked so strange to me 😆

Reply:
Done